lib/fetch/index.js

```diff
@@ -0,0 +1,88 @@
+import Prismic from 'prismic.io';
+import { PrismicConfig } from '../config/prismic.js';
+import {
+  sanitizeEvent,
+  sanitizeNews,
+  sanitizeCommunity,
+  sanitizeSpeaker,
+  sanitizeTalk,
+} from './sanitize';
+
+async function fetchOne(docType, sanitizeFunction, args) {
```

Comment:
I started to do this but the tests on our side are really opaque and hard to modify, so I reverted until I could work out the best way to rewrite the tests so that they are useful.

I actually really like the fact that the query and execution are two separate things, and queries are composable too :D 

Shame the limit doesn't seem to be considered part of the query though.